unittests/feed/LedgerFeedTests.cpp

```diff
@@ -127,10 +122,10 @@ TEST_F(FeedLedgerTest, AutoDisconnect)
     ctx.run();
     EXPECT_EQ(testFeedPtr->count(), 1);
 
+    EXPECT_CALL(*mockSessionPtr, send(_)).Times(0);
```

Comment:
Space missing. Also the comment is kinda useless - isn't the code self-speaking here?